amxmodx/srvcmd.cpp

```diff
@@ -63,6 +63,40 @@ void amx_command()
 
 		print_srvconsole("%d plugins, %d running\n", plugins, running);
 	}
+	else if (!strcmp(cmd, "plugin"))
+	{
+		auto id = atoi(CMD_ARGV(2));
+
+		if (CMD_ARGC() < 3)
+		{
+			print_srvconsole("Usage: amxx plugin < plugin_id >\nFor a list of plugins, use the \"amxx plugins\" command\n");
+		}
+		else
+		{
+			auto plugin = g_plugins.findPlugin(id);
+
+			if (plugin && plugin->isValid())
```

Comment:
if with init

Reply:
@In-line  ?

Reply:
Judging by the error, all the same AMX Mod X is not compiled in C ++17. I revert changes.